efficientdet/hparams_config.py

```diff
@@ -140,6 +166,7 @@ def default_detection_configs():
   # dataset specific parameters
   h.num_classes = 90
   h.skip_crowd_during_training = True
+  h.label_id_mapping = None
```

Comment:
How about make label_id_mapping as a json file name? (seems not easy to pass a huge dict to hparams).

Reply:
I like that a lot